src/reducers/initialReducer.js

```diff
@@ -0,0 +1,24 @@
+import { IS_REDUCING, HAS_REDUCED } from '../actions/types/initialTypes';
+
+const initialState = {
+  isReducing: 'false',
+};
+
+const initialReducer = (state = initialState, action) => {
+  switch (action.type) {
+    case IS_REDUCING:
+      return {
+        ...state,
+        isReducing: 'true',
+      };
+    case HAS_REDUCED:
+      return {
+        ...state,
+        isReducing: 'false',
```

Comment:
Why should this be `'false'` when it can be a boolean.

Reply:
I want it to show up in the DOM, if it's a boolean, it wont show